CMakeLists.txt

```diff
@@ -293,6 +275,58 @@ ELSE ()
 	ENDIF()
 ENDIF()
 
+IF (USE_HTTPS STREQUAL "ON")
+	IF (SECURITY_FOUND)
```

Comment:
It's a bit strange that we're actually searching for the security framewok even if `USE_HTTPS=OFF` here. But we can improve that later, no need to do it as part of this PR